flamingpy/cv/ops.py

```diff
@@ -28,6 +28,11 @@ def invert_permutation(p):
     return p_inverted
 
 
+def issparse(array):
+    """Check if an array is sparse. Backwards-compatible with old SciPy versions."""
+    return isinstance(array, getattr(sp, "sparray", sp.coo_matrix))
```

Comment:
this function is specific to this module. in scipy 0.10, the `SCZ_mat` function was producing a `scipy.sparse.coo_matrix`. In 0.13, it's producing a `csr_array`. I do this funky `getattr` thing because `csr_array` existed in 0.10 but I want something that didn't (because it _wasn't_ a `csr_array`).